ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -247,7 +248,18 @@ def __init__(
         self.config = config
         self.metadata_config = metadata_config
         self.metadata = OpenMetadata(metadata_config)
+
+        service = self.metadata.get_by_name(
+            entity=DashboardService, fqn=self.config.serviceName
+        )
+        if service:
+            self.config.serviceConnection = (
```

Comment:
It might be interesting to add a couple of lines explaining the behavior:
If the service exist already, we will override any incoming serviceConnection based on the information present in the backend, which is our source of truth

Reply:
Added a comment here https://github.com/nahuelverdugo/OpenMetadata/blob/6f80eb31e7a711c93a89a1eb4ca1dd5a8af27759/ingestion/src/metadata/ingestion/api/workflow.py#L81